notebooks/vertex_genai/solutions/gemini_prompt_engineering.ipynb

```diff
@@ -0,0 +1,952 @@
+{
```

Comment:
<p>The response is in Markdown not in JSON. Is it worth modifying the prompt to make sure the JSON is not included in Markdown? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Updated the prompt. Thanks! </p>